Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp

```diff
@@ -181,7 +181,8 @@ BackgroundMCCCollision::get_nu_max(amrex::Vector<MCCProcess> const& mcc_processe
         E_step = (energy_step < E_step) ? energy_step : E_step;
     }
 
-    for (amrex::ParticleReal E = E_start; E < E_end; E+=E_step) {
+    amrex::ParticleReal E = E_start;
+    while (E < E_end){
```

Comment:
Why that?

Reply:
This change was actually accepted in a different PR. The check that forces this is:
[cert-flp30-c](https://releases.llvm.org/14.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/cert-flp30-c.html) .
I acknowledge that the two versions of the code do exactly the same thing in this case, but I think that enforcing `cert-flp30-c` is generally a good idea.